modules/planning/open_space/trajectory_smoother/distance_approach_ipopt_fixed_ts_interface.cc

```diff
@@ -381,7 +381,7 @@ bool DistanceApproachIPOPTFixedTsInterface::get_starting_point(
   if (enable_constraint_check_) {
     int kM = m;
     double g[kM];
-    ADEBUG << "initial points constraint checking";
+    AINFO << "initial points constraint checking";
```

Comment:
Change to ADEBUG to mute redundant logs.

Reply:
> Change to ADEBUG to mute redundant logs.

fixed